### PR TITLE
feat: add test to test transactions with empty actions

### PR DESCRIPTION
**Pull request by heueristik**

### CODE REVIEW

File: contracts/test/ProtocolAdapterMock.t.sol [hunk 1]
```diff
@@ -175,6 +175,17 @@ contract ProtocolAdapterMockVerifierTest is Test {
         _mockPa.execute(txn);
     }
 
+    function test_execute_1_txn_with_up_to_3_empty_actions(bool[3] memory isEmpty) public {
```
[the remark this answers is not in the document]
Comment (heueristik) [reply contracts/test/ProtocolAdapterMock.t.sol:178]: This test is only about testing if an empty action being located in the beginning, middle, or end of the action array can cause troubles. This test only tests this one thing.
